services/wordpress/wordpress.tex

```diff
@@ -0,0 +1,26 @@
+\section{Wordpress}\index{wordpress}
+
+Wordpress is a PHP content management system. It has reasonable security in a new default install, but has a poor track record for remote execution exploits. The best way to secure Wordpress is to update it if possible and remove all unnecessary or old plugins. The following will help limit damage from possible remaining exploits:
+
+\begin{itemize}
+	\item make sure file permissions are good
+	\begin{itemize}
+		\item `/` needs to be writable only by user account
+		\item `/wp-content` needs to be writable by web server
+		\item `/wp-content/plugins` needs to be writable only by user account
+	\end{itemize}
+
+	\item remove unnecessary database permissions
+	\begin{itemize}
+		\item only allow the SQL user to have \lstinline|SELECT,INSERT,UPDATE,DELETE|
+	\end{itemize}
+
+	\item disable file editing from wp-admin
+	\begin{itemize}
+		\item add \lstinline|define('DISALLOW_FILE_EDIT', true);| to \lstinline|wp-config.php|
+	\end{itemize}
+
+	\item move \lstinline|wp-config.php| to the directory above the wordpress root
+
+	\item add 'AskApache Password Protect'
```

Comment:
Only .htaccess has to be enabled. I'm going to make it more specific, but AskApache Password Protect basically adds HTTP authentication to the admin panel preventing bugs there from being exploited.